src/taxonomy/import-tags/ImportTagsWizard.jsx

```diff
@@ -0,0 +1,374 @@
+// @ts-check
+import React, { useState, useContext } from 'react';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import {
+  Button,
+  Container,
+  Dropzone,
+  Icon,
+  IconButton,
+  ModalDialog,
+  Stack,
+  Stepper,
+} from '@edx/paragon';
+import {
+  DeleteOutline,
+  Download,
+  InsertDriveFile,
+  Warning,
+} from '@edx/paragon/icons';
+import PropTypes from 'prop-types';
+
+import LoadingButton from '../../generic/loading-button';
+import { getFileSizeToClosestByte } from '../../utils';
+import { TaxonomyContext } from '../common/context';
+import { getTaxonomyExportFile } from '../data/api';
+import { planImportTags, useImportTags } from './data/api';
+import messages from './messages';
+
+const linebreak = <> <br /> <br /> </>;
+
+const TaxonomyProp = PropTypes.shape({
+  id: PropTypes.number.isRequired,
+  name: PropTypes.string.isRequired,
+});
+
+const ExportStep = ({ taxonomy }) => {
+  const intl = useIntl();
+
+  return (
+    <Stepper.Step eventKey="export">
+      <Stack gap={3} data-testid="export-step">
+        <p>{intl.formatMessage(messages.importWizardStepExportBody, { br: linebreak })}</p>
+        <Stack gap={3} direction="horizontal">
+          <Button
+            iconBefore={Download}
+            variant="outline-primary"
+            onClick={() => getTaxonomyExportFile(taxonomy.id, 'csv')}
+            data-testid="export-csv-button"
+          >
+            {intl.formatMessage(messages.importWizardStepExportCSVButton)}
+          </Button>
+          <Button
+            iconBefore={Download}
+            variant="outline-primary"
+            onClick={() => getTaxonomyExportFile(taxonomy.id, 'json')}
+            data-testid="export-json-button"
+          >
+            {intl.formatMessage(messages.importWizardStepExportJSONButton)}
+          </Button>
+        </Stack>
+      </Stack>
+    </Stepper.Step>
+  );
+};
+
+ExportStep.propTypes = {
+  taxonomy: TaxonomyProp.isRequired,
+};
+
+const UploadStep = ({
+  file,
+  setFile,
+  importPlanError,
+  setImportPlanError,
+}) => {
+  const intl = useIntl();
+
+  /** @type {(args: {fileData: FormData}) => void} */
+  const handleFileLoad = ({ fileData }) => {
+    setFile(fileData.get('file'));
+    setImportPlanError(null);
+  };
+
+  const clearFile = (e) => {
+    e.stopPropagation();
+    setFile(null);
+    setImportPlanError(null);
+  };
+
+  return (
+    <Stepper.Step eventKey="upload" hasError={importPlanError}>
+      <Stack gap={3} data-testid="upload-step">
+        <p>{intl.formatMessage(messages.importWizardStepUploadBody, { br: linebreak })}</p>
+        <div>
+          {!file ? (
+            <Dropzone
+              style={{ height: '200px' }}
```

Comment:
@rpenido It seems like you are using this style in several places. maybe it's better to [create a utility](https://github.com/openedx/frontend-app-course-authoring/commit/dab9d8a01a0389826e989252b112c3b70869e251#diff-c2d8998567caf1b28fc98e349d9fdbe0b95a3e4ea15938b79b7ce147b90b9cbdR5). Ref: https://github.com/openedx/frontend-app-course-authoring/pull/723#discussion_r1425040465

Reply:
Thank you! FIxed here: 2d3abe8542acc16d49bbceb59aed9e1fe82e5bd7